BrightCoders-Institute/s10a1-proyecto-final-team-01-s10a1-proyecto-final
# Guests' Favorites Accommodations System

# Description

- Added an option in order to allow all the users to mark any accommodation as their favorite.
- Added a filter in accommodations index in order to only show the user's currently favorite accommodations.

## Testing

- Run the rails db:migrate to properly create the database in a local PostgreSQL environment.
- Install Action Text and Active Storage.
- Run the rails db:seed command in order to load all the default users data.
- Run either bin/dev or rails server and open the localhost:3000 endpoint in any browser.
- Login with any account in order to check the new accommodations' features.
